mesheryctl/pkg/utils/error.go

```diff
@@ -286,6 +297,12 @@ func formatError(msg string, cmd cmdType) string {
 		return fmt.Sprintf("%s\nSee %s for usage details\n", msg, modelViewURL)
 	case cmdRegistry:
 		return fmt.Sprintf("%s\nSee %s for usage details\n", msg, registryUsageURL)
+	case cmdWorkspace:
```

Comment:
As a separate question, what is the difference in the format/msg, except the parameters? we can just have a function that accepts the params and displays them.

Something to track and complete it another PR.